packages/perseus/src/widgets/__tests__/__snapshots__/interactive-graph.test.ts.snap

```diff
@@ -84,7 +84,7 @@ exports[`interactive-graph widget question Shoud render predictably: after inter
                     style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"
                   />
                   <path
-                    d="M371.42857142857144,228.57142857142858L142.85714285714286,57.142857142857146L142.85714285714286,228.57142857142858Z"
```

Comment:
Not a ship blocker, just curious: I wonder why this one is different?

Reply:
I figured out the difference by pasting these into an SVG path editor tool. Before, it was showing a right triangle (the answer), and now it's showing a default starting triangle.

This particular snapshot was looking at `polygonQuestion`, referenced in `questionsAndAnswers` in the testdata file.

This new triangle is actually the expected triangle. I'm not sure why it was showing a different triangle before, as the `graph` field in the testdata wasn't specifying that it should be the same as the `correct` coords. 🤔 The fact that it was wrong is weird, but at least it's fixed now?

| Before | After |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/b97186a5-969a-4d7a-ba84-b3724193a00a) | ![image](https://github.com/user-attachments/assets/71a2dce2-0080-4523-a5d2-4acf4fbe807a) |